requirements.txt

```diff
@@ -9,3 +9,6 @@ tomli==2.2.1; python_version <= '3.10'
 # hipBLASLt deps
 joblib>=1.4.2
 msgpack>=1.1.0
+
+# AWS Redshift connector
+redshift_connector==2.1.8
```

Comment:
Ah, redshift-connector isn't as lightweight as I hoped: https://pypi.org/project/redshift-connector/
```
Requires-Dist: scramp (<1.5.0,>=1.2.0)
Requires-Dist: pytz (>=2020.1)
Requires-Dist: beautifulsoup4 (<5.0.0,>=4.7.0)
Requires-Dist: boto3 (<2.0.0,>=1.9.201)
Requires-Dist: requests (<3.0.0,>=2.23.0)
Requires-Dist: lxml (<6.0.0,>=4.6.5)
Requires-Dist: botocore (<2.0.0,>=1.12.201)
Requires-Dist: packaging
Requires-Dist: setuptools
```

https://github.com/aws/amazon-redshift-python-driver/blob/master/requirements.txt

We may want to move that into a separate requirements file... or tell them to fix their install requirements :P